modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,107 @@
+
+.Prerequisites 
+
+* Installed the OpenShift CLI (`oc`). 
+
+.Procedure
+* Create a machine set and modify the `resourceID` and `vmSize` parameters. This machine set will control the `arm64` worker nodes in your cluster.
++
+[source,terminal]
+----
+oc create -f arm64-machine-set-0.yaml
+----
+.Sample YAML machine set with `arm64` bootimage 
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+    machine.openshift.io/cluster-api-machine-role: worker
+    machine.openshift.io/cluster-api-machine-type: worker
+  name: <infrastructure_id>-arm64-machine-set-0
+  namespace: openshift-machine-api
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-arm64-machine-set-0
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-arm64-machine-set-0
+    spec:
+      lifecycleHooks: {}
+      metadata: {}
+      providerSpec:
+        value:
+          acceleratedNetworking: true
+          apiVersion: machine.openshift.io/v1beta1
+          credentialsSecret:
+            name: azure-cloud-credentials
+            namespace: openshift-machine-api
+          image:
+            offer: ""
+            publisher: ""
+            resourceID: /resourceGroups/${RESOURCE_GROUP}/providers/Microsoft.Compute/galleries/${GALLERY_NAME}/images/rhcos-arm64/versions/1.0.0 <1>
+            sku: ""
+            version: ""
+          kind: AzureMachineProviderSpec
+          location: <region>
+          managedIdentity: <infrastructure_id>-identity
+          networkResourceGroup: <infrastructure_id>-rg
+          osDisk:
+            diskSettings: {}
+            diskSizeGB: 128
+            managedDisk:
+              storageAccountType: Premium_LRS
+            osType: Linux
+          publicIP: false
+          publicLoadBalancer: <infrastructure_id>
+          resourceGroup: <infrastructure_id>-rg
+          subnet: <infrastructure_id>-worker-subnet
+          userDataSecret:
+            name: worker-user-data
+          vmSize: Standard_D4ps_v5 <2>
+          vnet: <infrastructure_id>-vnet
+          zone: "<zone>"
+---- 
+<1> Set the `resourceID` parameter to the `arm64` bootimage.
+<2> Set the `vmSize` parameter to the instance type used in your installation. Some example instance types are `Standard_D4ps_v5` or `D8ps`.
+
+.Verification
+. Verify that the new ARM64 machines are running: 
++
+[source,terminal]
+----
+$ oc get machineset -n openshift-machine-api
```

Comment:
@aleskandro Should have asked this before. This command is for verifying the arm machines are running as well right?